Client/Frontend/Widgets/TwoLineImageOverlayCell.swift

```diff
@@ -0,0 +1,159 @@
+        containerView.addSubview(stackView)
+        containerView.addSubview(leftOverlayImageView)
+
+        contentView.addSubview(containerView)
+        bringSubviewToFront(containerView)
+
+        NSLayoutConstraint.activate([
+            containerView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            containerView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            containerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor,
+                                                    constant: -16),
+
+            leftImageView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor,
+                                                   constant: 16),
+            leftImageView.centerYAnchor.constraint(equalTo: containerView.centerYAnchor),
+            leftImageView.heightAnchor.constraint(equalToConstant: 28),
+            leftImageView.widthAnchor.constraint(equalToConstant: 28),
+            leftImageView.trailingAnchor.constraint(equalTo: stackView.leadingAnchor,
+                                                   constant: -16),
+
+            leftOverlayImageView.trailingAnchor.constraint(equalTo: leftImageView.trailingAnchor,
+                                                          constant: 8),
+            leftOverlayImageView.bottomAnchor.constraint(equalTo: leftImageView.bottomAnchor,
+                                                        constant: 8),
+            leftOverlayImageView.heightAnchor.constraint(equalToConstant: 22),
+            leftOverlayImageView.widthAnchor.constraint(equalToConstant: 22),
+
+            stackView.topAnchor.constraint(equalTo: containerView.topAnchor,
+                                           constant: 8),
+            stackView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor,
+                                              constant: -8),
+            stackView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor,
+                                                    constant: -8),
+        ])
+
+        selectedBackgroundView = selectedView
+        applyTheme()
+    }
+
+    func configure() {
+
+    }
+
+    func applyTheme() {
+        let theme = BuiltinThemeName(rawValue: LegacyThemeManager.instance.current.name) ?? .normal
+        if theme == .dark {
+            self.backgroundColor = UIColor.Photon.Grey80
+            self.titleLabel.textColor = .white
+            self.descriptionLabel.textColor = UIColor.Photon.Grey40
+            self.selectedView.backgroundColor = UIColor.theme.tableView.selectedBackground
+        } else {
+            self.backgroundColor = .white
+            self.titleLabel.textColor = .black
+            self.descriptionLabel.textColor = UIColor.Photon.DarkGrey05
+            self.selectedView.backgroundColor = UIColor.theme.tableView.selectedBackground
```

Comment:
nit: remove self?
Also, wondering if to this point we should start using the new theme manager in new code. This would help us migrate to new theme system. cc: @OrlaM

Reply:
I'd say leave it for the moment because we are still ironing out the odd kink in it. Let's aim for the next eng meeting to have it in good enough shape for everyone to use and we can do a presentation on how to use it.